Base/usr/bin/env

```diff
@@ -0,0 +1 @@
+/bin/env
```

Comment:
This should probably be a symlink to `/bin/env` rather than a shell script (without a shebang?)

```diff
$ gitd Meta/build-root-filesystem.sh 
diff --git a/Meta/build-root-filesystem.sh b/Meta/build-root-filesystem.sh
index 71785bb51..1814ca92c 100755
--- a/Meta/build-root-filesystem.sh
+++ b/Meta/build-root-filesystem.sh
@@ -117,6 +117,7 @@ echo "done"
 printf "installing shortcuts... "
 ln -s Shell mnt/bin/sh
 ln -s test mnt/bin/[
+ln -s /bin/env mnt/usr/bin/env
 echo "done"
 
 printf "installing 'checksum' variants... "
```

Reply:
This actually is a symlink. symlinks are just rendered weirdly on github.